R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -177,6 +177,13 @@ test_that("add and get file to be downloaded with Spark job on every node", {
   spark.addFile(path)
   download_path <- spark.getSparkFiles(filename)
   expect_equal(readLines(download_path), words)
+
+  # Test spark.getSparkFiles works well on executors.
+  seq <- seq(from = 1, to = 10, length.out = 5)
+  f <- function(seq) { readLines(spark.getSparkFiles(filename)) }
+  results <- spark.lapply(seq, f)
+  for (i in 1:5) { expect_equal(results[[i]], words) }
+
```

Comment:
hmm, that's odd. install.spark could have changed SPARK_HOME perhaps - we probably need to step through this to get more details debugging this. 
I'm fine if it passes runtests - I'm concerned if a path is returned but udf can't read the file (like the test you have `readLines(x)` - which should be the main use of this change, yes?